src/meshReaders/Hex_Mesh_Reader.hh

```diff
@@ -140,37 +140,32 @@ private:
  * <h3> Format Details </h3>
```

Comment:
It looks like most of the HTML (or whatever this is) was stripped out in the following lines, but not the <h3></h3> here -- why is that? Is this a clang-tidy oversight?

Reply:
I just pushed a fix for this.  Again - old code.

Reply:
(LOL, should've realized it would treat that as actual HTML :D )